src/utils.rs

```diff
@@ -152,3 +152,29 @@ pub(crate) fn parse_string(raw: Vec<u8>) -> MomentoResult<String> {
         details: None,
     })
 }
+
+pub trait IntoBytes: Send {
+    fn into_bytes(self) -> Vec<u8>;
+}
+
+impl<T: Send> IntoBytes for T
+where
+    T: Into<Vec<u8>>,
+{
+    fn into_bytes(self) -> Vec<u8> {
+        self.into()
+    }
+}
+
+pub trait IntoBytesIterable: Send {
+    fn into_bytes(self) -> Vec<Vec<u8>>;
+}
+
+impl<T: Send> IntoBytesIterable for Vec<T>
+where
+    T: IntoBytes,
+{
+    fn into_bytes(self) -> Vec<Vec<u8>> {
+        self.into_iter().map(|item| item.into_bytes()).collect()
+    }
+}
```

Comment:
Can you add some documentation to these traits since they appear in public function signatures?